src-java/kilda-pce/src/test/java/org/openkilda/pce/mapper/PathSegmentMapperTest.java

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.pce.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.openkilda.model.PathId;
+import org.openkilda.model.PathSegment;
+import org.openkilda.model.Switch;
+import org.openkilda.model.SwitchId;
+import org.openkilda.pce.Path.Segment;
+
+import com.google.common.collect.Sets;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+public class PathSegmentMapperTest {
+
+    @Test
+    public void toPathSegmentTest() {
+        Segment segment1 = Segment.builder()
+                .latency(12L)
+                .srcSwitchId(new SwitchId("00:01"))
```

Comment:
[minor] this switchId could be also initialized with an int new SwitchId(1)
```suggestion
                .srcSwitchId(new SwitchId(1)))
```

In any case, I would create the 3 switchIds previously